lcls-twincat-motion/Library/DUTs/DUT_MotionPneumaticActuator.TcDUT

```diff
@@ -0,0 +1,157 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<TcPlcObject Version="1.1.0.1" ProductVersion="3.1.4024.3">
+  <DUT Name="DUT_MotionPneumaticActuator" Id="{ee78164b-027c-4a52-afc0-269c58a42ceb}">
+    <Declaration><![CDATA[{attribute 'qualified_only'}
+{attribute 'strict'}
+TYPE DUT_MotionPneumaticActuator :
+    // Defines the EPICS interface to actuating a pneumatic stage
+STRUCT
+     (* Hardware *)
+    //Readbacks
+    //Limit Switch
+    {attribute 'pytmc' := '
+     pv: PLC:bInLimitSwitch
+     io: i
+     field: ZNAM FALSE
+     field: ONAM TRUE
+     field: DESC TRUE if IN limit is reached
+    '}
+    i_bInLimitSwitch    :    BOOL;
```

Comment:
What do the leading `i` and `q` mean in these struct variables? I thought they would be inputs/outputs but these don't have input/output links on them.

Reply:
The actual inputs and outputs are read into epics via these pvs. I don't pragma the actual i and q. the values are then assigned during the soft IO mapping. I keep the same terminology to indicate that these are the soft inputs and outputs.